code/style.css

```diff
@@ -1,4 +1,259 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* Mobile first*/
+body {
+  max-width: 1920;
+  padding: 10%;
+  
 }
+
+.full-container {
+  margin: 0;
+  padding: 10px 0px 10px 0px;
+}
+
+.navbar {
+width: 100%;
+overflow: auto;
+display: flex;
+}
+
+.navbar a {
+  font-size: 15px;
+  font-weight: bold;
+  color: black;
+  padding: 10px;
+  text-decoration: none;
+  overflow: hidden;
+  cursor: pointer;
+}
+
+.navbar a:hover {
+  background-color: #1F7980;
+  transition: 0.2s;
+  color: white;
+}
+
+.headline-section-grid {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 10px;
+  background: white;
+}
+
+.header-grid-container {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 10px;
+}
+
+.title-grid {
+  grid-column: span 3;
+  text-align: center;
+  display:inline-flex;
+  padding-top: 20px;
+}
+
+.bignews-grid1 {
+  grid-column: span 1;
+  width: 100%;
+}
+
+.bignews-grid2 {
+  grid-column: span 1;
+  display: block;
+  width: 100%;
+}
+
+.body-section-grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 15px;
+  background: white;
+}
+
+.smallnews-grid {
+  grid-column: span 4;
+  display: block;
+  width: 100%;
+  justify-content: center;
+
+}
+
+.body-section-grid img {
+  object-fit: cover;
+  border-radius: 5%;
+  height: 200px;
+  border:2px solid #fff;
+  background: url(img/tiger.png) no-repeat;
+  box-shadow: 10px 10px 5px #ccc;
+  -moz-box-shadow: 10px 10px 5px #ccc;
+  -webkit-box-shadow: 10px 10px 5px #ccc;
+  -khtml-box-shadow: 10px 10px 5px #ccc;
+  opacity: 1;
+  transition: .5 ease;
+  backface-visibility: hidden;
+
+}
+
+.cred {
+  font-size: 15px;
+}
+
+.grid-pic img {
+  animation-duration: 6s;
+  animation-iteration-count: infinite;
+  animation-name: logobounce;
+  animation-timing-function: cubic-bezier(0.280, 0.840, 0.420, 1);
+  margin: 10px;
+}
+
+@keyframes logobounce {
```

Comment:
nice effect and it is cool that you managed to add animation , I will also have to try it in the next project.